src/libs/elektra/key.c

```diff
@@ -478,29 +480,31 @@ static void keyClearNameValue (Key * key)
 /**
  * A destructor for Key objects.
  *
- * Every key created by keyNew() must be
+ * Every Key created by keyNew() must be
  * deleted with keyDel().
  *
- * It is safe to delete keys which are
- * in a keyset, the number of references
- * will be returned then.
+ * It is safe to delete Keys which are in a KeySet. Then the Key will not be deleted
+ * and the number of references will be returned.
  *
- * It is save to delete a nullpointer,
+ * It is safe to delete a NULL pointer,
  * -1 will be returned then.
  *
- * It is also save to delete a multiple
- * referenced key, nothing will happen
+ * It is also safe to delete a multiple
```

Comment:
Maybe this can be consolidated with "`Keys which are in a KeySet`" stuff. Seems like the effect is the same.